tests/scripts/all.sh

```diff
@@ -2296,7 +2302,8 @@ component_test_psa_crypto_config_accel_ecdh_use_psa () {
     # -------------
 
     msg "test: MBEDTLS_PSA_CRYPTO_CONFIG with accelerated ECDH"
-    make test
+    #make test
+    tests/ssl-opt.sh
```

Comment:
This looks like an error to me: I think that in this PR we should run `make test`, but not `ssl-opt.sh` as the latter is supposed to be solved in a following issue/PR. Isn't it?

Reply:
Good catch :)